src/Command/Redis/HSCAN.php

```diff
@@ -84,6 +84,15 @@ public function parseResponse($data)
         return $data;
     }
 
+    /**
+     * @param $data
+     * @return array|mixed|string|null
+     */
+    public function parseResp3Response($data)
```

Comment:
IMHO these should be private

Reply:
Then it's makes no sense. This method is called from outside in case if RESP3 is active and you need additional response processing for specific command. It works the same as already existing `parseResponse()` method